### DISCUSSION TITLE
Protected visibility for metadata components

### DISCUSSION THREAD

**Discussion by kbommaSFDC** [jongpie/NebulaLogger#285] [General]
**Configuration/objects/LoggerParameter__mdt** and **logger-framework/objects/LoggerPlugin__mdt** metadata components visibility is configured as Protected in the package but production org don't allow to deploy any components with Protected visibility.  We had to change the visibility to Public in order to deploy in prod successfully. 

Just wanted to let community know that without changing the visibility for these two components deployment would fail in production org.

**Comment by jongpie** [comment]
@kbomma-sf that's great info to have for anyone that's deploying the metadata (instead of using the unlocked or managed packages), thanks for sharing!
